src/activeobjectmgr.h

```diff
@@ -25,15 +25,16 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 class TestClientActiveObjectMgr;
 class TestServerActiveObjectMgr;
 
-template <typename T> class ActiveObjectMgr
+template <typename T>
+class ActiveObjectMgr
 {
 	friend class ::TestClientActiveObjectMgr;
 	friend class ::TestServerActiveObjectMgr;
 
 public:
 	virtual void step(float dtime, const std::function<void(T *)> &f) = 0;
-	virtual bool registerObject(T *obj) = 0;
-	virtual void removeObject(u16 id) = 0;
+	virtual bool registerObject(T *obj)								  = 0;
+	virtual void removeObject(u16 id)								  = 0;
```

Comment:
What a monstrosity. Thanks clang-format.

Reply:
changing a setting has such stupid side effects

Reply:
So what do you do here to de-align this, insert an empty line?